roles/maas_pacemaker/tasks/main.yaml

```diff
@@ -0,0 +1,39 @@
+---
+- name: "Install Pacemaker packages"
+  ansible.builtin.apt:
+    name: "{{ item }}"
+    state: present
+    update-cache: true
+  with_items:
+    - pacemaker
+    - pcs
+    - fence-agents
+
+- name: "Ensure Pacemaker is started"
+  ansible.builtin.systemd:
+    name: "pacemaker.service"
+    state: started
+    enabled: false # if the system is rebooted, the user may not want this member to rejoin automatically
+
+- name: "Check if fencing is configured"
+  ansible.builtin.shell:
+    cmd: "set -opipefail; pcs stonith status | tee {{ maas_pacemaker_tmp_status }}"
+    creates: "{{ maas_pacemaker_tmp_status }}"
+  register: pacemaker_stonith_status
+
+- name: "Dump existing CIB"
+  ansible.builtin.command:
+    cmd: "pcs cluster cib {{ maas_pacemaker_tmp_cib }}"
+    creates: "{{ maas_pacemaker_tmp_cib }}"
+  when: pacemaker_stonith_status.output == "NO stonith devices configured"
```

Comment:
After the previous task I have some trouble getting this task to run, but it might be resolved if the previous task gets fixed. I am not sure if .output is correct- perhaps it needs to be .stdout-- unsure. Maybe 'NO .." needs to be "No .." (capitalisation), or something else?